File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/GeoPosition.java
package com.example.testlunu.net.acu.cityinfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class GeoPosition {

    @SerializedName("Latitude")
    @Expose
    public Float latitude;
    @SerializedName("Longitude")
    @Expose
    public Float longitude;
    @SerializedName("Elevation")
    @Expose
    public Elevation elevation;

}

File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/GeoPosition.kt
package com.example.testlunu.net.acu.cityinfo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class GeoPosition (
    @SerializedName("Latitude")
    @Expose
    var latitude: Float?,
    @SerializedName("Longitude")
    @Expose
    var longitude: Float?,
    @SerializedName("Elevation")
    @Expose
    var elevation: Elevation?
)